src/cryptography/hazmat/backends/openssl/backend.py

```diff
@@ -1391,10 +1391,10 @@ def derive_elliptic_curve_private_key(self, private_value, curve):
 
         point = self._lib.EC_POINT_new(group)
         self.openssl_assert(point != self._ffi.NULL)
-        point = self._ffi.gc(point, self._lib.EC_POINT_free)
+        point = self._ffi.gc(point, self._lib.EC_POINT_clear_free)
```

Comment:
See here.

Reply:
Mind you that this is also a public value. Other than that, not sure if I see a use for `EC_POINT_clear_free` if I'm going off of the logic from the implementation of `BN_clear_free`.